### PR TITLE
Calculator logic

### PR DESCRIPTION
**Pull request by lather10**

### CODE REVIEW

File: calculator/calc.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+using namespace std;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: stop the usage of `using namespace std;`

File: calculator/calc.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+int main()
+{
+    double a,b,c;
+    int n;
+
+    cout<<"\n$$$ ENTER NUMERICAL VALUES ONLY TO PERFORM OPERATION AS IT IS CALCULATOR: $$$"<<endl;
+    cout<<"----------------------------------------------------------------------------------------\n";
+
+
+    cout<<"Enter First Number"<<endl;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: Ask for user's choice first and then ask for input numbers

File: calculator/calc.cpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    cout<<"You have chosen option: "<<"***"<<n<<"***";
+    //cout<<"----------------------------------------------------------------------------------------\n";
+    cout<<" And Here is the result printed below: \n"<<endl;
+    while(n!=5)
+    {
+    switch(n)
+    {
+    case 1:
+        cout<<"Addition of numbers is: ";
+        c=a+b;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: move this arithmetic operation to a separate function

File: calculator/calc.cpp [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        cout<<c<<endl<<endl;
+        break;
+    case 2:
+        cout<<"Subtraction of numbers is: ";
+        c=a-b;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: same as above, move this arithmetic operation to a separate function

File: calculator/calc.cpp [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        cout<<c<<endl<<endl;
+        break;
+
+    case 3:
+        cout<<"Multiplication of numbers is: ";
+        c=a*b;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: same as above, move this arithmetic operation to a separate function

File: calculator/calc.cpp [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        cout<<c<<endl<<endl;
+        break;
+
+    case 4:
+
+        cout<<"Division of numbers is: ";
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: same as above, move this arithmetic operation to a separate function

File: calculator/calc.cpp [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+        c=a/b;
```
Comment (VeerOfficial) [inline calculator/calc.cpp]: this code will break if value of b is 0. Think about that corner case and add protection

**Review by VeerOfficial — COMMENTED** [review]
Good effort but it still needs some work. 
Try to address the below review comments and then I'll take a look again

File: calculator/calc.cpp (hunk 2 above)
Comment (lather10) [reply calculator/calc.cpp:12]: Resolved

File: calculator/calc.cpp (hunk 1 above)
Comment (lather10) [reply calculator/calc.cpp:2]: resolved

Comment (lather10) [reply calculator/calc.cpp]: resolved

**Comment by VeerOfficial** [comment]
Good work. For now you can merge this code to main branch.
Only minor suggestion is that try to take input of numbers in main only and then call the respective function as per user choice.

**Pull request merged**